index.html

```diff
@@ -1,20 +1,219 @@
+    height: 120vh;
+    width: 30%;
+    object-fit: cover;
+    opacity: 0.2;
+     }
+   .about-info{
+    color: #fff;
+    opacity: 0.6;S
+    line-height: 40px;
+    font-size: 30px;
+       } 
+   .h1-section{
+      text-decoration:underline;
+       text-shadow:2px 1px yellow;
+     } 
+  </style>
+  </head>
+  <body>
+  <nav class="navbar">
+     <!-- LOGO -->
+     <div class="logo"><img src ="GDlogo.jpg" alt="Geeta Turumella" width=35px></div>
+     <!-- NAVIGATION MENU -->
+     <ul class="nav-links">
+       <!-- USING CHECKBOX -->
+       <input type="checkbox" id="checkbox_toggle" />
+       <label for="checkbox_toggle" class="hamburger">&#9776;</label>
+       <!-- NAVIGATION MENUS -->
+       <div class="menu">
+  <li class="link-li"><a href="#About">About</a></li>
+  <li class="link-li"><a href="#Project">Project</a></li>
+  <li class="link-li"><a href="#Experience">Experience</a></li>
+  <li class="link-li"><a href="#Connect">Connect</a></li>
+  </div>
+  </ul>
+  </nav>
+
+  <!-- ABOUT SECTION-->	
+          <section class="about-section active">
+          <h2 class="hero-heading"><marquee style="direction:right">hello! i am<br> geeta</marquee></h2>
+		  <img src="geeta.png" class="about-img" alt=" ">
+	      <div class="about-info" id="about">
+		  		  <p>Thanks for checking out my profile! I am re-enteering the workforce after 12 years. I have a Master's degree, Diploma in Advanced Computer Technology and Web Designing. 
+				  I am a person with versatile skillset, integrity and willingness to learn new skills. I am curious, driven, and always looking to improve my abilities. I am looking for entry level web development or returnship jobs.  I am confident in my ability to successfully succeed in any task that I take up. </p>
+        </div>
+	             </section>
+   
+  
+		<!--Experience section-->
+        <section> 
+		  <div>
+                   <h1 class="h1-section">Experience</h1>
+           <ul>
+        <li>2021:<br>
+         <p>Certified in Going Places Network, a <strong>Dress for success program</strong> for unemployed and under employed women from <strong>Waketech Community College</p>
+            </li><br> <br>
+			
+            <li>2010-2022: 
+            <p>I have been out of work for the past 12 years as I was taking care of home and my son.  I have been doing some volunteering work in my son’s school as well as chaperoning the field trips before pandemic.  I am actively managing home, looking after my son as well as self teaching myself on current trends in web developer job market.</p><br><br>
+        </li>
+    
+            <li>
+            <p> 2009-2010: <br> Worked as a junior level developer, Broadgate Inc, Troy, MI</p>
+        </li>
+          </ul>
+        </div>
+ </section>
+ &nbsp &nbsp
+ <section>
+ <h1 class="h1-section">Connect</hi>
+  <h3> Email: gtturumella@gmail.com &nbsp GitHub
+  <a href="https://github.com/Tugeeta"><img src="github.jpg" width="15"> &nbsp &nbsp Linkedin</a>
```

Comment:
character entities should have a semi-colon: `&nbsp;`... but if you need to change the padding, probably better to do that with css.